exercises/chapter4/test/no-peeking/Solutions.purs

```diff
@@ -41,7 +41,7 @@ keepNonNegativeRewrite :: Array Number -> Array Number
 keepNonNegativeRewrite arr = (\n -> n >= 0.0) <$?> arr
 
 isPrime :: Int -> Boolean
-isPrime n = eq 1 $ length $ factors n
+isPrime n = (n > 1) && (eq 1 $ length $ factors n)
```

Comment:
```suggestion
isPrime n = n > 1 && length (factors n) == 1
```
If you want to save some parens. Not sure if this makes things less clear.

Reply:
Yeah, this is preferable in my view. A linter (if one existed) would probably complain about what you have here.

Reply:
Because of the redundant parens, that is